### DISCUSSION TITLE
Размер репозитория

### DISCUSSION THREAD

**Discussion by Z-A-E-B-A-L** [community/community#51671] [Codespaces]
### Select Topic Area

Question

### Body

Какой максимальный размер репозитория?

**Comment by ramazancetinkaya** [comment]
@Z-A-E-B-A-L,

You can check this link:

https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github

--- 

Hope this clarifies.

If you find this reply useful, please kindly 'Mark as answer' so that others know this issue is already addressed.

Have a good day.

**Comment by samruddhikhandale** [comment]
Hi 👋 

> We recommend repositories remain small, ideally less than 1 GB, and less than 5 GB is strongly recommended

https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github#repository-size-limits also contains more information.
